src/pybind/mgr/volumes/module.py

```diff
@@ -320,6 +320,24 @@ class Module(orchestrator.OrchestratorClientMixin, MgrModule):
             'desc': "Cancel an pending or ongoing clone operation.",
             'perm': 'r'
         },
+        {
+            'cmd': 'fs subvolume snapshot mirror enable '
```

Comment:
You expect these 2 commands to be used instead of `fs snapshot mirror add/remove` going forward?